repos/spack_repo/builtin/packages/rivet/package.py

```diff
@@ -49,6 +49,9 @@ class Rivet(AutotoolsPackage):
         description="HepMC version to link against",
     )
 
+    variant("plugin-match", default="none", description="List of Rivet analyses to be included")
+    variant("plugin-unmatch", default="none", description="List of Rivet analyses to be excluded")
```

Comment:
Functionality only added in 4.1.0; https://gitlab.com/hepcedar/rivet/-/merge_requests/957.
```suggestion
    with when("@4.1:"):
        variant("plugin-match", default="none", description="List of Rivet analyses to be included")
        variant("plugin-unmatch", default="none", description="List of Rivet analyses to be excluded")
```